### ISSUE TITLE
UITabBarController animation and default animations

### ISSUE THREAD

**Issue by leonardo-ferreira07** [lkzhao/Hero#113]
Hey @lkzhao, first of all, thank you for this amazing library! 

- Is there anything that we can set to change UITabBarController animation when user is going to another ViewController from UITabBarController, and new ViewController have the `hidesBottomBarWhenPushed` set to `true`? Actually, by default, Hero slide the TabBar to left, we would like to slide it to bottom. We're doing that from our code, but would like to knows if is possible do it with Hero.

- We are using your lib in version 0.1.1 until today. Today when we've updated it, we saw the default animations overriding the `HeroID` transitions, like when pushing new ViewControllers, the default push animation is acting together with `HeroID` elements in root and destination ViewController, how we can disable this defaults animations introduced in v0.3.0?

We have searched in docs about this two questions, but didn't find anything. Thanks for your help!

**Comment by lkzhao** [comment]
1. Can you show me an video or example project? Is the other view controller inside the UITabBarController? Or is it being pushed by a navigation controller?

2. Set the `heroNavigationAnimationType` to `.fade` on the navigation controller will disable the behaviour

**Comment by leonardo-ferreira07** [comment]
Thank you for your answer and sorry for the delay, @lkzhao.

1. I've recorded a video to show you better what's happening. The view controllers stack is like: TabBarController -> NavigationController -> ViewController with the list where I selected the element and goes to another screen. If you see in video, the tabbar disappears and in the presented screen it show for a second and then disappears again, that's the bug.

[tabbar-bug.zip](https://github.com/lkzhao/Hero/files/906656/tabbar-bug.zip)

Thank you so much!
